src/coprocessor/statistics/analyze.rs

```diff
@@ -234,6 +234,7 @@ impl<S: Snapshot> SampleBuilder<S> {
             ranges,
             req.take_primary_column_ids().into(),
             false,
+            false,
```

Comment:
Maybe we should add some comments here to emphasize that we don't support streaming mode in `Analyze` request.